benchmark/benchmark.c

```diff
@@ -164,143 +196,150 @@ static inline void seL4_BenchmarkTrackDumpSummary(benchmark_track_kernel_entry_t
 
 void notified(microkit_channel ch)
 {
-    switch(ch) {
-        case START:
-            sel4bench_reset_counters();
-            THREAD_MEMORY_RELEASE();
-            sel4bench_start_counters(benchmark_bf);
+    switch (ch) {
+    case START:
+        sel4bench_reset_counters();
+        THREAD_MEMORY_RELEASE();
+        sel4bench_start_counters(benchmark_bf);
 
-            #ifdef CONFIG_BENCHMARK_TRACK_UTILISATION
-            microkit_benchmark_start();
-            #endif
+#ifdef CONFIG_BENCHMARK_TRACK_UTILISATION
+        microkit_benchmark_start();
+#endif
 
-            #ifdef CONFIG_BENCHMARK_TRACK_KERNEL_ENTRIES
-            seL4_BenchmarkResetLog();
-            #endif
+#ifdef CONFIG_BENCHMARK_TRACK_KERNEL_ENTRIES
+        seL4_BenchmarkResetLog();
+#endif
 
-            break;
-        case STOP:
-            sel4bench_get_counters(benchmark_bf, &counter_values[0]);
-            sel4bench_stop_counters(benchmark_bf);
+        break;
+    case STOP:
+        sel4bench_get_counters(benchmark_bf, &counter_values[0]);
+        sel4bench_stop_counters(benchmark_bf);
 
-            sddf_printf("{\n");
-            for (int i = 0; i < ARRAY_SIZE(benchmarking_events); i++) {
-                sddf_printf("%s: %lX\n", counter_names[i], counter_values[i]);
-            }
-            sddf_printf("}\n");
+        sddf_printf("{\n");
+        for (int i = 0; i < ARRAY_SIZE(benchmarking_events); i++) {
+            sddf_printf("%s: %lX\n", counter_names[i], counter_values[i]);
+        }
+        sddf_printf("}\n");
+
+#ifdef CONFIG_BENCHMARK_TRACK_UTILISATION
```

Comment:
Can this code be moved into a separate function with no body if CONFIG_BENCHMARK_TRACK_UTILISATION is undef?  and get rid of the ifdef here?